packages/pn-personagiuridica-webapp/src/components/Contacts/CourtesyContactItem.tsx

```diff
@@ -154,7 +154,8 @@ const CourtesyContactItem = ({ recipientId, type, value, blockDelete }: Props) =
           ref={digitalElemRef}
           fields={[
             {
-              id: `value-${type}`,
+              id: `courtesyContacts-${type}`,
+              key: 'courtesyContactValue',
```

Comment:
Not use static key. Use `courtesyContactValue-${type}`

Reply:
done